django_stomp/services/consumer.py

```diff
@@ -71,7 +73,9 @@ def __init__(
     def _create_new_worker_executor(self):
         return ThreadPoolExecutor(max_workers=STOMP_PROCESS_MSG_WORKERS, thread_name_prefix=self._subscription_id)
 
-    def on_message(self, headers, body):
+    def on_message(self, frame: StompFrame):
```

Comment:
**curiosity:** The `on_message` method signature has changed on newer `stomp.py` versions?

Reply:
Yep! Not only `on_message`, but almost every `on_*` had a change in their signature (check out the current [`ConnectionListener` class](https://github.com/jasonrbriggs/stomp.py/blob/ed12801cd8128003ff5cf55534fb3c3dba43c302/stomp/listener.py#L48)). The change was made in two commits ([eba178b](https://github.com/jasonrbriggs/stomp.py/commit/eba178b5dda5cab873a4b45ac040587a9f55b7f3#diff-3e0179888d23f0b6afd11ab21000fdcdc73f0aeb2a3e5e55b774c019c32fc96b) and [2481924](https://github.com/jasonrbriggs/stomp.py/commit/24819247acede4e3429f0d7746b0d1133a96081f#diff-3e0179888d23f0b6afd11ab21000fdcdc73f0aeb2a3e5e55b774c019c32fc96b)) that were released in version `6.1.0`

Reply:
Nice, my friend! Thanks for the clarification!